packages/next-admin/src/components/MainLayout.tsx

```diff
@@ -37,7 +37,7 @@ export const MainLayout = ({
     >
       <I18nProvider translations={mergedTranslations}>
         <ColorSchemeProvider>
-          <div className="next-admin__root h-[100vh]">
+          <div className="next-admin__root">
```

Comment:
We can remove this className which was some legacy stuff originated from the dark mode implementation